data/strings/strings.txt

```diff
@@ -29890,49 +29890,142 @@
+    fi = Organic Maps tarvitsee sijaintitietoja. Kun se on turvallista, tarkista puhelimesi ilmoitus.
+    fr = Organic Maps a besoin d'un accès à la localisation. Lorsque c'est sûr, vérifie la notification sur ton téléphone.
+    he = מפות אורגניות זקוקות לגישה למיקום. כשזה בטוח, בדוק את ההתראה בטלפון שלך.
+    hi = ऑर्गेनिक मैप्स को स्थान पहुंच की आवश्यकता है। जब यह सुरक्षित हो, तो अपने फ़ोन पर अधिसूचना जांचें।
+    hu = Az Organic Maps-nek szüksége van a helymeghatározáshoz való hozzáférésre. Ha biztonságos, ellenőrizze az értesítést a telefonján.
+    id = Organic Maps membutuhkan akses lokasi. Jika sudah aman, periksa notifikasi di ponsel Anda.
+    it = Organic Maps ha bisogno di accedere alla posizione. Quando è sicuro, controlla la notifica sul tuo telefono.
+    ja = オーガニック・マップは位置情報へのアクセスを必要とする。安全になったら、携帯電話の通知をチェックしよう。
+    ko = 오가닉 지도에는 위치 액세스 권한이 필요합니다. 안전해지면 휴대폰에서 알림을 확인합니다.
+    lt = Organic Maps reikia prieigos prie vietos. Kai bus saugu, patikrinkite pranešimą telefone.
+    mr = ऑर्गेनिक नकाशेला स्थान प्रवेश आवश्यक आहे. ते सुरक्षित असताना, तुमच्या फोनवरील सूचना तपासा.
+    nb = Organic Maps trenger posisjonstilgang. Når det er trygt, kan du sjekke varslingen på telefonen.
+    nl = Organic Maps heeft locatietoegang nodig. Als het veilig is, controleer je de melding op je telefoon.
+    pl = Aplikacja Organic Maps wymaga dostępu do lokalizacji. Gdy będzie to bezpieczne, sprawdź powiadomienie w telefonie.
+    pt = O Organic Maps precisa de acesso à localização. Quando for seguro, verifica a notificação no teu telemóvel.
+    pt-BR = O Organic Maps precisa de acesso à localização. Quando for seguro, verifique a notificação em seu telefone.
+    ro = Organic Maps are nevoie de acces la locație. Când este sigur, verificați notificarea de pe telefon.
+    ru = Organic Maps нужен доступ к местоположению. Когда будет безопасно, проверьте уведомление на телефоне.
+    sk = Služba Organic Maps potrebuje prístup k polohe. Keď je to bezpečné, skontrolujte oznámenie na telefóne.
+    sv = Organic Maps behöver platsåtkomst. När det är säkert kan du kontrollera aviseringen på din telefon.
+    sw = Organic Maps zinahitaji ufikiaji wa eneo. Ikiwa salama, angalia arifa kwenye simu yako.
+    th = แผนที่ทั่วไปจำเป็นต้องเข้าถึงตำแหน่ง เมื่อปลอดภัยแล้ว ให้ตรวจสอบการแจ้งเตือนบนโทรศัพท์ของคุณ
+    tr = Organic Maps konum erişimine ihtiyacı vardır. Güvenli olduğunda, telefonunuzdaki bildirimi kontrol edin.
+    uk = Для роботи Organic Maps потрібен доступ до місцезнаходження. Коли це буде безпечно, перевірте сповіщення на телефоні.
+    vi = Bản đồ không phải trả tiền cần quyền truy cập vị trí. Khi an toàn, hãy kiểm tra thông báo trên điện thoại của bạn.
+    zh-Hans = 有机地图需要位置访问权限。安全后，查看手机上的通知。
+    zh-Hant = 有機地圖需要位置存取權限。當安全時，請查看手機上的通知。
+
+  [aa_request_permission_notification]
+    comment = Notification title for permission request from AA.
+    tags = android
+    en = This app needs your permission
```

Comment:
How will it look?
Maybe something like:
```suggestion
    en = Location permission is required
```

Or `Please grant location permission`? "Your permission" sounds a bit strange.

Reply:
I took this string from Google Maps